documentation/css/scrollbar.css

```diff
@@ -0,0 +1,20 @@
+*::-webkit-scrollbar {
+    width: 16px;
```

Comment:
use rem

Reply:
Since this is a scrollbar, I thought to keep the size constant.